backend/benefit/applications/api/v1/serializers/application.py

```diff
@@ -567,6 +575,12 @@ def get_former_benefit_info(self, obj):
             "last_possible_end_date": last_possible_end_date,
         }
 
+    def get_ahjo_decision_date(self, obj):
```

Comment:
Hmm... batch will be removed later on when Ahjo integration is done. Works for now, but this will have to be revisited later.